framework/source/class/qx/bom/request/Xhr.js

```diff
@@ -1062,7 +1062,7 @@ qx.Bootstrap.define("qx.bom.request.Xhr",
       if (this._getProtocol() === "file:") {
         error = !this.responseText;
       } else {
-        error = !this.statusText && this.status !== 204;
+        error = this.status === 0;
```

Comment:
Does this change really improve compatibility? Or does it only improve conformance to RFC7230, but maybe leads to failing clients which do not conform to RFC7230. Conformance does not allways mean compatibility. Compatibility is something we may also assure for non conforming clients.

But in the end I fear that I'm not familiar enough with Xhr, so others have to give their opinion on this PR.